### PR TITLE
Slider for price filtering when searching in listings

### PR DESCRIPTION
**Pull request by cathykc**
(sorry should have created new branch for this)

### CODE REVIEW

File: app/static/styles/app.css [hunk 1]
```diff
@@ -28,12 +28,15 @@ table.selectable tr:hover {
...
-.add_quant {
-	float: left;
-	width: 70%;
+#input_size {
+  width: 60px;
 }
+
+#slider-range{
+  margin-top:10px;
+}
```
Comment (jondubin) [inline app/static/styles/app.css]: Nit: add a new line

File: app/templates/merchant/view_listings.html [hunk 2]
```diff
@@ -34,16 +33,19 @@
...
-                <input value="{{ min_price }}" type="text" name="min-price" placeholder="e.g. $0">
-            </div>
-            <div class="three wide field">
-                <label>Maximum unit price</label>
-                <input value="{{ max_price }}" type="text" name="max-price" placeholder="e.g. $10">
+        <div class="twelve wide field advanced-search" style="{{ display }}">
+            <div class="two wide fields">
+                <div class="three wide field">
+                    <label>Minimum unit price</label>
+                    <input value="{{ min_price }}" type="number" id="min-price" name="min-price" placeholder="e.g. $0">
```
Comment (jondubin) [inline app/templates/merchant/view_listings.html]: We should coerce the numbers we're showing here into having two digits after the decimal point

Comment (cathykc) [inline app/templates/merchant/view_listings.html]: To indicate that it's a price? Or do you want the slider to be specific enough to select within dollars.

Comment (jondubin) [inline app/templates/merchant/view_listings.html]: Just to indicate it's a price

**Pull request merged**